src/dataservice/data.go

```diff
@@ -0,0 +1,330 @@
+
+package main
+
+import (
+        "bytes"
+        "encoding/json"
+        "flag"
+        "fmt"
+        "io/ioutil"
+        "math/rand"
+        "net/http"
+        "net/url"
+        "strconv"
+        "time"
+
+        "github.com/vmware/harbor/src/common/models"
+        "github.com/manveru/faker"
+)
+
+// Harbor deployment URL
+var baseURL string
+
+// This instantiates a Faker module used to generate randomized data
```

Comment:
AFAIK there are no provisions to control the data generated. It has limited provisions to tweak the data generated.
For now, we use Faker to create the Project name and the user details i.e. name, username etc